app/uk/gov/hmrc/uknwauthcheckerapi/errors/ApiErrorResponse.scala

```diff
@@ -95,6 +95,12 @@ final case class UnauthorizedApiError(reason: String) extends ApiErrorResponse {
   val message:    String = ApiErrorMessages.unauthorized
 }
 
+case object RequestEntityTooLargeError extends ApiErrorResponse {
```

Comment:
Please move this into the correct position in the file. It's currently alphabetically sorted.